### PR TITLE
closes #21 | add tailor fitting model

### LINKED ISSUE #21: Add Tailor rating model
Consult https://app.genmymodel.com/edit/_5uMicFMlEeeTnI9B59buBQ#

**PS**: Also try to use the npm module `shortid` for the `_id` field instead of the deafult `mongoid`.

### PR DESCRIPTION
**Pull request by demiladebamgbose**
*Description of Task*
-  Add mongoose model for tailor fitting

### CODE REVIEW

**Comment by bolorundurowb** [comment]
@adetola-ralph you are to review this

**Comment by bolorundurowb** [comment]
@demiladebamgbose please update this branch from develop.

@adetola-ralph please review as soon as she does

File: src/server/models/tailorRating.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  comment: {
+    type: String
+  },
+  fittingPunctality: {
+    type: Number
+  },
+  fittingService: {
+    type: Number
+  },
+  delievery: {
```
Comment (adetola-ralph) [inline src/server/models/tailorRating.js]: Wrong spelling, should be `delivery`

File: src/server/models/tailorRating.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  },
+  title: {
+    type: String,
+    require: true,
+    unique: true
+  },
+  comment: {
+    type: String
+  },
+  fittingPunctality: {
```
Comment (adetola-ralph) [inline src/server/models/tailorRating.js]: Same here `punctuality`

File: src/server/models/tailorRating.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+const tailorRatingSchema = new Schema({
+  orderId: {
+    type: Schema.ObjectId,
+    ref: 'Order',
+    required: true
+  },
+  title: {
+    type: String,
+    require: true,
```
Comment (adetola-ralph) [inline src/server/models/tailorRating.js]: The property should be `required` not `require`

File: src/server/models/tailorRating.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import mongoose from 'mongoose';
+
+const Schema = mongoose.Schema;
+
+const tailorRatingSchema = new Schema({
+  orderId: {
+    type: Schema.ObjectId,
```
Comment (adetola-ralph) [inline src/server/models/tailorRating.js]: @bolorundurowb I thought we were going with `shortid` here and type `string`

**Comment by bolorundurowb** [comment]
@adetola-ralph Yes, the type is `String` and the default is `shortid.generate` cc: @demiladebamgbose

**Pull request merged**